hooks/tk-nuke/render_media.py

```diff
@@ -200,7 +202,7 @@ def __create_output_node(self, path):
 
         # apply any additional knob settings provided by the hook. Now that the knob has been
         # created, we can be sure specific file_type settings will be valid.
-        for knob_name, knob_value in wn_settings.iteritems():
+        for knob_name, knob_value in six.iteritems(wn_settings):
```

Comment:
I'm not sure how many knobs there would be on a node, so I played it safe and tried to keep the original behaviour in Python 2.